### PR TITLE
Changed conllu2json to be able to extract NER tags

### PR DESCRIPTION
**Pull request by katarkor**
## Description
conllu2json now checks the 10th column of the first word: if it's a NER tag it will extract NER tags from the whole conllu file and add them to the json file. If not, it creates the json file as before, without NER tags.

## Checklist

- [x] I have submitted the spaCy Contributor Agreement.
- [ ] I ran the tests, and all new and existing tests passed.
- [ ] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by katarkor** [comment]
Did I do it correctly? I'm not very good with git, sorry.

**Comment by honnibal** [comment]
Honestly I think git isn't very good at git :(. I'm constantly confused too. I'm not sure why these previous commits have gotten dragged in, but I think it won't matter.

Thanks for your help!

**Comment by odinho** [comment]
@katarkor First of all thanks for all of your contributions, that's really awesome. I don't know much about language tech, but Git I know very well.

The error you've done is that you did not create a new branch with your changes. You should always update your own local repository from this so-called upstream repository. When you do a new feature, change or bugfix, you should create a new branch for that change.

    $ git remote add upstream https://github.com/explosion/spaCy/  # adding a new remote "upstream"
    $ git fetch  # update your local repo
    $ git branch my_new_feature upstream/master
    $ git checkout my_new_feature
    $ # hack hack hack
    $ git commit -m"My cool feature" spacy/nb/changed_file.py
    $ git push origin my_new_feature:my_new_feature # push your branch to your own GitHub repo
    # do the manual pull request from `my_new_feature` branch on your fork, to `master` on `explosion` repository

If you already have done the change you want on the master branch, you can create a new branch as above, and get your changes into the new branch by cherry-picking them (rebasing is quicker, but cherry-pick easier to understand):

     $ git cherry-pick 2678ee8 # commit 1
     $ git cherry-pick 63c25c9 # commit 2 (have to be done in order)

Basically, you want a new branch for every change you want to open a pull request for.

**Comment by katarkor** [comment]
@odinho Thanks for explanation, I didn't really use git much before now. I tried to do what you proposed, but I got 
fatal: Not a valid object name: 'upstream/master'.
when I run "git branch my_new_feature upstream/master". Any idea why it happens and how to fix it? When I run "git remote" I get "origin" and "upstream", so it's there.

**Comment by odinho** [comment]
Great :)  It could have been that you had indeed added the remote, but not downloaded the data from it. So a `git fetch` or `git fetch upstream` would have fixed that, if that was the problem.

Git has rather good documentation, albeit a bit verbose:
https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging
